src/inventory/inventory.py

```diff
@@ -19,33 +21,89 @@ def __init__ (self):
         else:
             self.__checkForUpdates()
 
-    # TODO... Implement
-    def add_card(self, card):
-        self.connection.commit()
-        return False
-
-    # TODO... Implement
-    def remove_card(self, card):
-        self.connection.commit()
-        return False
+   # TODO: Add count values to database to track cards contained in inventory
```

Comment:
The headers that represent a variant of a card (foil, nonfoil, full_art, etc..) have been set up to track the stock count of that particular variant, for every card in the database. 

Adding/Removing a card will need to specify which one of these pre-exisiting headers to update.

Reply:
Totally forgot that's how we did it. I'll add that in a seperate branch.